vignettes/reuse.Rmd

```diff
@@ -110,27 +110,104 @@ add <- function(x, y) x + y
+
+By supplying a space separated list of components after the function name, you can also choose to inherit only selected components. For example, `@inherit foo return` would just inherit the `@return` tag, and `@inherit foo seealso source` would inherit the `@seealso` and `@source` tags.
+
+`@inherit foo sections` will inherit _every_ `@section` tag (which can also be specified in markdown by using the top-level heading spec, `#`). To inherit a _specific_ section from another function, use `@inheritSection foo Section title`. For example, all the "adverbs" in purrr use `#' @inheritSection safely Adverbs` to inherit a standard section that provides advice on using an adverb in package code.
+
+### Documenting `...`
+
+When your function passes `...` on to another function, it can be useful to inline the documentation for some of the most common arguments. This technique is inspired by the documentation for `plot()`, where `...` can take any graphical parameter; `?plot` describes some of the most common arguments so that you don't have to look them up in `?par`.
+
+`@inheritDotParams` has two components: the function to inherit from and the arguments to inherit. Since you'll typically only want to document the most important arguments, `@inheritDotParams` it comes with a flexible specification for argument selection inspired by `dplyr::select()`:
+
+-   `@inheritDotParams foo` takes all parameters from `foo()`
+-   `@inheritDotParams foo a b e:h` takes parameters `a`, `b`, and all parameters between `e` and `h`
+-   `@inheritDotParams foo -x -y` takes all parameters except for `x` and `y`.
+
+### Inheriting from other packages
+
+It's most common to inherit from other documentation topics within the current package, but roxygen2 also supports inheriting documentation from other packages by using `package::function` syntax, e.g.:
+
+-   `@inheritParams package::function`
+
+-   `@inherit package::function`
+
+-   `@inheritSection package::function Section title`
 
-    -   `@inheritDotParams foo` takes all parameters from `foo()`
-    -   `@inheritDotParams foo a b e:h` takes parameters `a`, `b`, and all parameters between `e` and `h`
-    -   `@inheritDotParams foo -x -y` takes all parameters except for `x` and `y`.
+-   `@inheritDotParams package::function` 
 
-All of these tags also work to inherit documentation from functions in another package by using `pkg::source_function`.
+When inheriting documentation from another package bear in mind that you're now taking a fairly strong dependency on an external package, and now to ensure every developer produces the same documentation you'll need to make sure that they all have the same version of the package installed. For that reason, this technique is best used sparingly.
```

Comment:
Maybe also worth saying something like, if the upstream package removes that function, section, or topic page, it can cause a downstream warning in your package (we've had to carefully avoid this in vctrs and dplyr sometimes when renaming topics https://github.com/tidyverse/dplyr/blob/b359331a448a693546d77245b0de4d405bab3886/R/utils-tidy-eval.R#L31